pkgs/test/texlive/default.nix

```
@@ -381,16 +381,16 @@ rec {
       shortVersion = [ "devnag" "diadia" "pmxchords" "ptex2pdf" "simpdftex" "ttf2afm" ];
       # (1) binaries requiring --help or -h
       help = [ "arlatex" "bundledoc" "cachepic" "checklistings" "dvipos" "extractres" "fig4latex" "fragmaster"
-        "kpsewhere" "latex-git-log" "ltxfileinfo" "mendex" "perltex" "pn2pdf" "psbook" "psnup" "psresize" "purifyeps"
+        "kpsewhere" "latex-git-log" "ltxfileinfo" "mendex" "pdflatexpicscale" "perltex" "pn2pdf" "psbook" "psnup" "psresize" "purifyeps"
         "simpdftex" "tex2xindy" "texluac" "texluajitc" "upmendex" "urlbst" "yplan" ];
-      shortHelp = [ "adhocfilelist" "authorindex" "bbl2bib" "bibdoiadd" "bibmradd" "biburl2doi" "bibzbladd" "ctanupload"
+      shortHelp = [ "adhocfilelist" "authorindex" "bbl2bib" "bibdoiadd" "bibmradd" "biburl2doi" "bibzbladd" "bookshelf-mkfontsel" "ctanupload"
```

The better treatment for `bookshelf-listallfonts` is the same as `bookshelf-mkfontsel`:
```suggestion
      shortHelp = [ "adhocfilelist" "authorindex" "bbl2bib" "bibdoiadd" "bibmradd" "biburl2doi" "bibzbladd" "bookshelf-listallfonts" "bookshelf-mkfontsel" "ctanupload"
```
Likewise for `ignoreExitCode`. That way it does not try to call `luaotfload-tool`.

---

Did you test that? IIRC when I tried that, it would not work (no such option, or something). If command runs without the arguments and correctly checks for its dependency `luaotfload-tool` that's a better test, right?

---

Yes, `bookshelf-listallfonts -h` seems to work fine here. If you wish to test that it works with `luaotfload-tool`, then `needScheme` is more appropriate as it will test the wrapped binary with the dependencies in path, but the `binaries` test doesn't require that you also check functionality (there are too many binaries!).

---

Done. Also had to add to `ignoreExitCode`.